src/logic/documents/scorecards.js

```diff
@@ -329,7 +332,15 @@ const scorecard = ({
   printStations,
   scorecardPaperSize,
   featured = false,
+  language = 'en',
 }) => {
+  const defaultTranslation = translation('en');
+  const translationData = translation(language);
+
+  const t = stringKey => {
+    return translationData[stringKey] || defaultTranslation[stringKey];
+  };
```

Comment:
You can do something like this:

```suggestion
  const defaultTranslationData = translation("en");
  const translationData = translation(language);
  
  const t = (...keys) => {
    const [phrase, defaultPhrase] = keys.reduce(
      [translationData, defaultTranslationData],
      ([data1, data2]) => [data1[key], data2[key]],
    );
  
    return phrase || defaultPhrase;
  };
```

So that we can conveniently query nested keys, such as `t('eventName', eventId)` and fallbacks work at any level automatically.